kubernetes-deployment/bootstrap.py

```diff
@@ -395,9 +397,33 @@ def remove_nodes(cluster_config, node_list_config):
 
 
 
+def maintain_expert(cluster_config, maintain_config, node_config, job_name):
+
+    module_name = "maintainlib.{0}".format(job_name)
+    module = importlib.import_module(module_name)
+
+    job_class = getattr(module, job_name)
+    job_instance = job_class(cluster_config, node_config, False)
+
+    job_instance.run()
+
+
+
+def maintain_nodes(cluster_config, node_list_config, job_name):
+
+    # Todo: load maintain from a DB such as etcd instead of a yaml file.
+    maintain_config = load_yaml_file("maintain.yaml")
+
+    for host in node_list_config['machinelist']:
+
+        maintain_expert(cluster_config, maintain_config, node_list_config['machinelist'][host], job_name)
```

Comment:
>maintain_expert [](start = 8, length = 15)

renamed to maintain_one_node?

Reply:
For easy understand?

Reply:
yes. maintain_expert is hard to understand. maintain_one_node is clear semantically

---
In reply to: [164986808](https://github.com/Microsoft/pai/pull/281#discussion_r164986808) [](ancestors = 164986808)

Reply:
Resolved.